pilot/main.py

```diff
@@ -60,7 +62,6 @@ def init():
                                 f"{'' if len(app['development_steps']) == 0 else app['development_steps'][-1]['id']:3}"
                                 f"  {app['name']}" for app in get_created_apps_with_steps()))
 
-            run_exit_fn = False
```

Comment:
if get_created_apps_with_steps() throws exception then we will not set run_exit_fn = False so we will run exit fn when we shouldnt